apps/web/components/Organization/Slice/SliceMachine.tsx

```diff
@@ -133,6 +139,10 @@ const renderSlice = (slice, namespace, slug, params) => {
       return webRenderConnectedComponent(slice)
     case 'FeaturedSupportQNAs':
       return <FeaturedSupportQNAs slice={slice} />
+    case 'connectedComponent':
```

Comment:
Do we have both `connectedComponent` and `ConnectedComponent` slice types (see `case 'ConnectedComponent':` above)?

Reply:
You are right, what I've done is essentially unnecessary